keps/sig-cloud-provider/0002-cloud-controller-manager.md

```diff
@@ -79,11 +79,8 @@ Controlled by [publishing kubernetes-rules-configmap](https://github.com/kuberne
 
 ## Summary
 
-We want to remove any cloud provider specific logic from the kubernetes/kubernetes repo. We want to restructure the code
-to make it easy for any cloud provider to extend the kubernetes core in a consistent manner for their cloud. New cloud
-providers should look at the [Creating a Custom Cluster from Scratch](https://kubernetes.io/docs/getting-started-guides/scratch/#cloud-provider)
-and the [cloud provider interface](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go#L31)
-which will need to be implemented.
+This KEP outlines the architectural changes required to run the Cloud Controller Manager. This is part of a long-running effort
```

Comment:
At some point I believe we need to expand the scope. CCM is a mechanism for allowing cloud specific controllers to be run in an K8s approved manner. I feel that our goal though is to provide a mechanism to allow every cloud provider (and in fact non cloud providers as well) to create installable Kubernetes which run correctly on their configurations. We would like to do that without including any cloud provider code in the main repo. To that end the CCM is necessary but not sufficient and we need to start tracking the additional pieces.